test/WebJobs.Script.Tests/NodeEndToEndTests.cs

```diff
@@ -712,6 +712,28 @@ public void ExcludedFunction_NotAddedToHost()
             await t;
         }
 
+        [Fact]
+        public async Task PromiseResolve()
+        {
+            JObject input = new JObject
+            {
+                { "scenario", "promiseResolve" }
+            };
+
+            Task t = Fixture.Host.CallAsync("Scenarios",
+                new Dictionary<string, object>()
+                {
+                    { "input", input.ToString() }
+                });
+
+            await TestHelpers.Await(() =>
+            {
+                return t.IsCompleted;
+            }, timeout: 5000, pollingInterval: 1000);
+
+            await t;
```

Comment:
This is not currently needed (will change when the comment above is addressed), since it will only be reached if the task is completed (`TestHelpers.Await` will throw if the task never transitions)